resources/translations/de_DE/travel/retraction_combing.md

```diff
@@ -0,0 +1,46 @@
+Combing-Modus
+====
+Beim Combing wird vermieden, die Wände des Drucks zu überqueren, wenn man von einer Stelle zur anderen fährt. Das Überqueren der Wände hinterlässt in der Regel eine Narbe auf der Oberfläche, wenn die heiße Düse aus dem Druck austritt oder eintritt, was im Allgemeinen unerwünscht ist.
```

Comment:
Correct me if I'm wrong here, but I thought the word "man" could only really apply to people, not 3D printers? That could be a cultural thing though. We have a similar word "men" in Dutch which wouldn't be used when it's about a machine, similar to the English word "one": "when one travels from one place to another".

Reply:
you are right...
better 
Beim Combing wird vermieden, die Wände des Drucks zu überqueren, wenn der Drucker von einer Stelle zur anderen fährt.